qiskit_experiments/curve_analysis/curve_analysis.py

```diff
@@ -36,7 +39,60 @@ class CurveAnalysis(BaseCurveAnalysis):
     The fit parameters from the series defined under the analysis class are all shared
     and the analysis performs a single multi-objective function optimization.
 
-    See :class:`BaseCurveAnalysis` for overridable method documentation.
+    Subclass may override these methods to customize the fit workflow.
+
+    .. rubric:: _run_data_processing
+
+    This method performs data processing and returns the processed dataset.
+    By default, it internally calls :class:`DataProcessor` instance from the analysis options
+    and processes experiment data payload to create Y data with uncertainty.
+    X data and other metadata are generated within this method by inspecting the
+    circuit metadata. The series classification is also performed by based upon the
+    matching of circuit metadata and :attr:`SeriesDef.filter_kwargs`.
+
+    .. rubric:: _format_data
+
+    This method consumes the processed dataset and outputs the formatted dataset.
+    By default, this method takes the average of y values over
+    the same x values and then sort the entire data by x values.
+
+    .. rubric:: _generate_fit_guesses
+
+    This method creates initial guesses for the fit parameters.
+    This might be overridden by subclass.
```

Comment:
Since this is true of all the methods in this section, it doesn't need to be said for this one again.

```suggestion
```